examples/cmd/snapshotting/example_demo.go

```diff
@@ -301,6 +301,8 @@ func loadSnapshotSSH(ctx context.Context, socketPath, memPath, snapPath, ipToRes
 
 	socketFile := fmt.Sprintf("%s.load", socketPath)
 	cfg := createNewConfig(socketFile, withNetworkInterface(networkInterface))
+	cfg.KernelImagePath = ""
+	cfg.MachineCfg = models.MachineConfiguration{}
```

Comment:
This looks weird. Perhaps making the kernel image path and machine config an option would be more clear.